### ISSUE TITLE
Blueshift suicide ordenance experience

### ISSUE THREAD

**Issue by Namwty** [Skyrat-SS13/Skyrat-TG#24647]
## Reproduction:

Set up BZ, realise that injectors both on freezing chambers and burn chambers are no longer connected to atmos monitoring console, cuase why not. ( Same as birdshit, but everyone expected it on it )

meaning you have to go into the chamber itself to manualy enable injectors, which can cause risk of dying, and it's just not needed experience for anyone + consistency issue.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/104979184/e2748754-fa6b-4202-84e6-5d68dba7c318)

In short the console should be detecting injectors, and they should be accessable and tweakable trough said console, same as most maps ( apart from birdshot and it's freezing chamber ).
